test/tools/wallet.ts

```diff
@@ -0,0 +1,21 @@
+import { randomBytes } from 'crypto'
+
+/**
+ * Generates a random Ethereum-like wallet address
+ * @returns A random address starting with '0x' followed by 40 hex characters
+ */
+export function generateRandomWalletAddress(): string {
+  // Generate 20 bytes (40 hex characters) of random data
+  const randomBytesBuffer = randomBytes(20)
+  // Convert to hex string and ensure it starts with '0x'
+  return '0x' + randomBytesBuffer.toString('hex')
+}
+
+/**
+ * Generates multiple random Ethereum-like wallet addresses
+ * @param count Number of addresses to generate
+ * @returns Array of random addresses
+ */
+export function generateRandomWalletAddresses(count: number): string[] {
+  return Array.from({ length: count }, () => generateRandomWalletAddress())
+}
```

Comment:
Not for now, but we could move this to https://github.com/decentraland/platform-server-commons, so we don't start copying this utility to other places. WDYT?

Reply:
Changed it!